sql/telemetry/fennec_ios_events_v1/view.sql

```diff
@@ -0,0 +1,138 @@
+CREATE OR REPLACE VIEW
+    `moz-fx-data-shared-prod.telemetry.fennec_ios_events_v1` AS
+WITH base_events AS (
+
+SELECT
+  *,
+  DATE(submission_timestamp) AS submission_date,
+  event.f0_ AS timestamp,
+  event.f0_ AS event_timestamp,
+  event.f1_ AS event_category,
+  event.f2_ AS event_method,
+  event.f3_ AS event_object,
+  event.f4_ AS event_value,
+  event.f5_ AS event_map_values,
+  metadata.uri.app_version,
+  osversion AS os_version,
+  metadata.geo.country
+FROM
+  telemetry.mobile_event
+  CROSS JOIN UNNEST(events) AS event
+
+), all_events AS (
+SELECT
+    submission_date,
+    submission_timestamp,
+    client_id AS device_id,
+    `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'session_id') AS session_id_offset,
+    CONCAT(document_id, CAST(timestamp AS STRING)) AS insert_id,
```

Comment:
`insert_id` is used for deduplication, so in this particular case we might lose some events (because of the unnest step above).

Assembling this exactly as in the [current implementation](https://github.com/mozilla/telemetry-streaming/blob/b85cdffc72a6f9ab224f9eececc38dfa09d98b8c/src/main/scala/com/mozilla/telemetry/pings/Ping.scala#L335-L338) would help with testing.